### PR TITLE
chore: pending review feedback for token details related changes

### PR DESCRIPTION
**Pull request by nikoferro**
Changes included for asset overview view

- Avoid trying to convert the price to selected currency if there is no fiat price reference available
- Add matching gradient to the area chart
- Make chart size bigger in relation to the rest of the asset overview
- Better gesture handling when interacting with the chart
- Subtle shadow on asset overview footer to match other parts of the app
- Change name from ETHER to Ethereum for mainnet's native currency

**Screenshots/Recordings**

screen

### CODE REVIEW

File: app/components/Views/Asset/index.js [hunk 1]
```diff
@@ -497,7 +505,7 @@ class Asset extends PureComponent {
           />
         )}
         {!asset.balanceError && (
-          <View style={styles.footer}>
+          <View style={{ ...styles.footer, ...styles.footerBorder }}>
```
Comment (NicolasMassart) [inline app/components/Views/Asset/index.js:508]: Q: I'm not sure what this this spread `...` does here with styles. Can you explain please?

Comment (leotm) [reply app/components/Views/Asset/index.js:508]: ^ should be equivalent to the style array format

```tsx
<View style={[styles.footer, styles.footerBorder }]>
```

so just applying `footer` style first
then applying (overriding) with `footerBorder` style

> last style in the array has precedence, so you can use this to inherit styles

Comment (nikoferro) [reply app/components/Views/Asset/index.js:508]: ☝️  @NicolasMassart pretty much what @leotm said. on the app it seems we use both approaches (the array and the spread)

**Review by jpcloureiro — APPROVED** [review]
code lgtm

**Review by leotm — APPROVED** [review]

**Pull request closed without merging**